NEWS.md

```diff
@@ -34,6 +34,11 @@ Outstanding ones only.
       You need to use a Hash literal to set a Hash to a first member.
       [[Feature #16806]]
 
+* Queue
+
+    * Queue#initialize now accepts an Enumerable of initial values.
+      [[Feature #17327]]
```

Comment:
You need to add `[Feature #17327]: https://bugs.ruby-lang.org/issues/17327` at the end of this file.

Reply:
Thanks done.